core/bones/date.py

```diff
@@ -17,6 +17,7 @@ def __init__(
         date: bool = True,
         localize: bool = False,
         time: bool = True,
+        tzNaive: bool = False,
```

Comment:
Please change this for PEP8-compliance, into something like
```suggestion
        naive: bool = False,
```